.golangci.yml

```diff
@@ -50,3 +50,34 @@ linters:
     - nonamedreturns
     - exhaustruct
     - gofumpt
+    - depguard
```

Comment:
I add several rules just ot disable a huge amount of cli linter errors.
Seems it would be better to open new task to make final changes related to linter